includes/admin/class-admin.php

```diff
@@ -1162,6 +1179,42 @@ private function check_deprecation_warnings() {
             }
         }
 
+		/**
+		 * Check old inline fontawesome setting.
+		 * If it was enabled by default, we show an admin notice.
+		 * On dismiss we keep the setting.
+		 *
+		 * @since x.x
+		 */
+		private function check_font_awesome_setting() {
+			if ( ! current_user_can( 'install_plugins' ) || ! WPBDP_App_Helper::is_bd_page() ) {
+				return;
+			}
+
+			if ( ! wpbdp_get_option( 'enqueue-fontawesome-styles', false ) ) {
+				return;
+			}
+
+			if ( is_plugin_active( 'font-awesome/index.php' ) ) {
+				$this->messages[] = array(
+					__( 'Good news! Business Directory Plugin now integrates with the official Font Awesome plugin.', 'business-directory-plugin' ),
+					'notice dismissible',
+					array( 'dismissible-id' => 'fa_dismiss' ),
+				);
+				return;
+			}
+
+			$this->messages[] = array(
+				sprintf(
+					__( 'Good news! Business Directory Plugin now integrates with the official Font Awesome plugin. %1$sInstall Font Awesome now%2$s.', 'business-directory-plugin' ),
+					'<a href=" ' . esc_url( admin_url( 'plugin-install.php?s=fontawesome&tab=search&type=author' ) ) . ' " target="_blank" rel="noopener nofollow">',
```

Comment:
I think we either need to not open this in a new tab, or make the link trigger the dismissal. Right now, when you click it, go install, and come back, the message is still there.

Reply:
I've made a change where on click, the notice will be dismissed, it wont open in a new tab. The notice will not show again other than in the Appearance tab in settings